org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/issues/SonarLintReportView.java

```diff
@@ -61,7 +61,7 @@ public void resetDefaultText() {
     if (reportTitle != null) {
       bottomLabel.setText(reportTitle + " (at " + DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm").format(reportDate) + "). <a>Learn more</a>");
     } else {
-      bottomLabel.setText("Run the analysis from the SonarLint context menu to find issues in the SCM change set or in all your project files. <a>Learn more</a>");
+      bottomLabel.setText("Run the analysis from the SonarQube context menu to find issues in the SCM change set or in all your project files. <a>Learn more</a>");
```

Comment:
```suggestion
      bottomLabel.setText("Run the analysis from the SonarQube for Eclipse context menu to find issues in the SCM change set or in all your project files. <a>Learn more</a>");
```

Reply:
No, because the context menu is actually just labeled `SonarQube`